src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -14,18 +14,13 @@ define([
     mixins: [BackboneMixin],
 
     propTypes: {
-      collection: React.PropTypes.object.isRequired,
-      onSelectApp: React.PropTypes.func.isRequired
+      collection: React.PropTypes.object.isRequired
     },
 
     getResource: function() {
       return this.props.collection;
     },
 
-    onClickApp: function(app) {
```

Comment:
Probably also remove this from AppComponent here: https://github.com/mesosphere/marathon/pull/990/files#diff-c2422f280116c15ea426918185c2d9b4R61